packages/client/src/__tests__/types/types.test.ts

```diff
@@ -58,6 +58,7 @@ describe('valid types', () => {
 async function runTsd(dir: string) {
   const diagnostics = await tsd({
     cwd: dir,
+    typingsFile: 'index.d.ts',
```

Comment:
`tsd` wants to import `default.d.ts` but our test is via `index.d.ts` in that folder.